test-definitions/fern/api/definition/__package__.yml

```diff
@@ -0,0 +1,37 @@
+types:
```

Comment:
probably a good thing to separate into 2 PRs since we sometimes have to revert stuff when things break (and I think these are unrelated?)

Reply:
True. Noted. They're slightly related. It's a fern definition playground that helped us test the error response examples.